### PR TITLE
CI: Quieten CI output.

### PR DESCRIPTION
**Pull request by jodh-intel**
Produce less output by default for some of the CI tests.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by dlespiau** [comment]
A fine question: will those script grow enough that we'll need to rewrite this in Go before the end?

lgtm

**Comment by jodh-intel** [comment]
We should atleast `bats` them soon. Is there such a thing as `/bin/gosh` yet? :smile:

**Comment by sboeuf** [comment]
lgtm

File: .ci/run.sh [hunk 1]
```diff
@@ -30,24 +30,27 @@ sudo docker exec ${container_id} echo hello
 sudo docker ps -a
 
 # try to unpause a container that isn't paused a few times
+tmpfile=$(mktemp)
 for i in 1 2 3
 do
-	{ sudo docker unpause ${container_id}; ret=$?; } || true
-	[ $ret -eq 1 ]
+	{ sudo docker unpause ${container_id} >"$tmpfile" 2>&1; ret=$?; } || true
```
Comment (devimc) [inline .ci/run.sh]: I guess ``` >"$tmpfile" 2>&``` is equal to ```&>"$tmpfile";  ```

File: .ci/run.sh [hunk 2]
```diff
@@ -30,24 +30,27 @@ sudo docker exec ${container_id} echo hello
...
+	[ $ret -eq 1 ] || { cat "$tmpfile"; exit 1; }
```
Comment (devimc) [inline .ci/run.sh]: Q. does docker always returns ```1``` on failure? wouldn't be better validate against ```0``` ?

Comment (devimc) [inline .ci/run.sh]: are you expecting ```docker unpause``` fails?

Comment (devimc) [inline .ci/run.sh]: [ $ret -eq 0 ] && { cat "$tmpfile"; exit 1; }

File: .ci/run.sh (hunk 1 above)
Comment (jodh-intel) [inline .ci/run.sh]: Yep - old habits die hard ;) As this is a bash script and not a Bourne shell one, fixed.

File: .ci/run.sh (hunk 2 above)
Comment (jodh-intel) [inline .ci/run.sh]: Good catch - fixed.

**Comment by devimc** [comment]
lgtm

**Pull request merged**